include/zephyr/dt-bindings/gpio/microchip-xec-gpio.h

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (c) 2022 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_MICROCHIP_XEC_GPIO_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_MICROCHIP_XEC_GPIO_H_
+
+/**
+ * @brief Microchip XEC GPIO bank and bit position convenience defines
+ *
+ * Microchip XEC documentation uses octal GPIO pin
+ * numbering. These macros do not require the user to do octal arithmetic
+ * to derive the pin's bit postion.
+ *
+ * Example DT usage: gpios = <&MCHP_GPIO176 GPIO_ACTIVE_HIGH>;
+ *
+ * @{
+ */
+
+#define XEC_PASTER(x, y, z)	x y z
```

Comment:
minor point: on a second look, I think this should be named XEC_GPIO_HELPER. "Pasting" in the C preprocessor almost always refers to usage of the `##` operator, which this is not using, so the name is misleading in my opinion. This seems reasonable to me though.

Reply:
updated to use XEC_GPIO_HELPER